sqlmesh/dbt/context.py

```diff
@@ -43,10 +44,22 @@ class DbtContext:
 
     _jinja_environment: t.Optional[Environment] = None
 
+    _manifest: t.Optional[ManifestHelper] = None
+
     @property
     def dialect(self) -> str:
         return self.engine_adapter.dialect if self.engine_adapter is not None else ""
 
+    @property
+    def manifest(self) -> ManifestHelper:
+        if self._manifest is None:
+            raise ConfigError("Manifest is not set in the context.")
```

Comment:
Is it a config error or sqlmesh error? My thinking no manifest means we screwed up, so it should be an internal error.